src/backend/starknet.cairo

```diff
@@ -295,4 +304,42 @@ namespace Internals {
 
         return _save_storage(starknet_address, storage_start + DictAccess.SIZE, storage_end);
     }
+
+    func _save_valid_jumpdests{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
so this is like `dict_keys` filtered for values > 0, I think that you can refactor this similar to the dict_keys util. Then, call the `write_jumpdests`.